### PR TITLE
Code cleaning and function description

### PR DESCRIPTION
**Pull request by elmarceloc**

### CODE REVIEW

File: app/services/chat.ts [hunk 1]
```diff
@@ -17,6 +17,12 @@ import { assertIsDefined } from 'util/properties-type-guards';
 import * as remote from '@electron/remote';
 import { SourcesService } from 'app-services';
 
+/**
+ * Enables the twitch's Better TTV emotes
```
Comment (andycreeth) [inline app/services/chat.ts:21]: Technically this function doesn't actually enable BTTV, it just returns the script that can be injected to enable it.  I think this comment could lead to confusion unless we update it to reflect that.

**Review by andycreeth — CHANGES_REQUESTED** [review]
Hi, thank you for your interest in contributing to Streamlabs Desktop.  I've done a quick review of your submission, and found a minor issue.  Thank you.

File: app/services/chat.ts (hunk 1 above)
Comment (elmarceloc) [reply app/services/chat.ts]: Thanks for the feedback

**Review by andycreeth — APPROVED** [review]

**Pull request merged**